helix-term/src/commands.rs

```diff
@@ -4403,7 +4405,9 @@ fn surround_add(cx: &mut Context) {
 
 fn surround_replace(cx: &mut Context) {
```

Comment:
Surround-replace has two `cx.on_next_key`s, one for the starting character and one for the replacement. Nested in the first `cx.on_next_key`, we add the character from the event to the pseudo-pending and then clear it within the second

Reply:
Thanks, I'll take a look at that.